test/jasmine/tests/parcoords_test.js

```diff
@@ -615,7 +615,7 @@ describe('@gl parcoords', function() {
             function restyleDimension(key, setterValue) {
 
                 // array values need to be wrapped in an array; unwrapping here for value comparison
-                var value = Lib.isArray(setterValue) ? setterValue[0] : setterValue;
+                var value = Array.isArray(setterValue) ? setterValue[0] : setterValue;
```

Comment:
Why did you need to do this?

Reply:
Because `Lib.isArray` is no more. It was replaced by `Lib.isArrayOrTypedArray` and `Lib.isTypedArray`. Here, these parcoord test don't use typed arrays, so good old `Array.isArray` suffice.